modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -13,10 +13,26 @@ import * as Options from './Options';
 import { createUploader, openNotification } from './util/ImageUploader';
 
 /**
+ * TinyMCE Editor Upload API
  * Handles image uploads, updates undo stack and patches over various internal functions.
  *
- * @private
  * @class tinymce.EditorUpload
+ * @example
+ * // Apply a filter to all images as uploaded..
```

Comment:
```suggestion
 * // Apply a filter to all images as they are uploaded.
```
1. Typo correction: s/.././
2. Suggested copy-edit: s/images as uploaded/images as they are uploaded/

Reply:
Correcting with reference to scan, rather than upload